src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx

```diff
@@ -25,10 +28,15 @@ const heading = (
 
 const requiredWarningHeading = (
   <>
-    {heading}
+    <VaAlert uswds status="warning">
+      <h2>
+        You submitted your CHAMPVA Other Health Insurance Certification form
+        without required documents
+      </h2>
+    </VaAlert>
     <p>
       You’ll still need to send us these required documents in order for us to
-      process this application:
+      process this form:
     </p>
```

Comment:
Updates this to show a yellow warning when the user has not submitted all documents alongside their form.